### ISSUE TITLE
pci_init() in v20.04 conflicts with pci_init() from libpci (PCI utilities)

### ISSUE THREAD

**Issue by mpiszczek** [spdk/spdk#1407]
Hello,

I recently upgraded a project to SPDK v20.04. The function pci_init(void) in lib/env_dpdk/pci.c now conflicts with pci_init() from the libpci PCI Utilities library.

## Possible Solution

Revert commit 15d0ae628de05172da695261c49864a40cef101f
lib/env_dpdk: remove spdk prefix from internal functions. 

## Steps to Reproduce

Compile this example code with the spdk and libpci
```
#include <pci/pci.h>
int main(int argc, char *argv[])
{
    struct pci_access *pci_acc = pci_alloc();
    pci_init(pci_acc);
    pci_scan_bus(pci_acc);
}
```

## Context (Environment including OS version, SPDK version, etc.)

SPDK version v20.04.1

pciutils-libs-3.5.1-3.el7.x86_64
pciutils-devel-3.5.1-3.el7.x86_64

Linux 4.15.0-88-generic #88-Ubuntu SMP Tue Feb 11 20:11:34 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

**Comment by jimharris** [comment]
We need to include a fix for this in v20.04.2 (tentatively planned for next week).

Moving to bug.

**Comment by tomzawadzki** [comment]
Proposal for resolution sent here: https://review.spdk.io/gerrit/c/spdk/spdk/+/2611
Along with backport to v20.04.2: https://review.spdk.io/gerrit/c/spdk/spdk/+/2614
